tests/LSP7DigitalAsset/extensions/LSP7Mintable.behavior.ts

```diff
@@ -57,7 +58,7 @@ export const shouldBehaveLikeLSP7Mintable = (
         context.accounts.tokenReceiver.address
       );
 
-      expect(tokenReceiverBalance.toNumber()).toEqual(amountToMint.toNumber());
+      expect(tokenReceiverBalance.toNumber()).to.be.equal(amountToMint.toNumber());
```

Comment:
The chai matchers from Hardhat now work with BN, so no need anymore to convert `.toNumber()`

```suggestion
      expect(tokenReceiverBalance).to.be.equal(amountToMint);
```